lib/datasource/remote/firebase/firebase_remote_config.dart

```diff
@@ -9,6 +9,7 @@ const String _defaultEndPointUrlKey = 'default_end_point';
 const String _defaultV2EndPointUrlKey = 'default_v2_end_point';
 const String _featureFlagImportAccount = 'feature_flag_import_account';
 const String _featureFlagExportRecoveryPhrase = 'feature_flag_export_recovery_phrase';
+const String _featureFlagDelegate = 'feature_flag_delegate';
```

Comment:
you are missing the default value. 
Look at the rest of the code in this file

Reply:
I also think the name needs the enabled word to clarify the fact that if true it is enabled

Reply:
Agree

Reply:
Fix